packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -202,7 +202,18 @@ export class Interval implements ISerializableInterval {
     }
 }
 
-export class SequenceInterval implements ISerializableInterval {
+/**
+ * ISequenceIntervalEvents events should only be used internally in IntervalCollections.
```

Comment:
```suggestion
 * @internal
 * ISequenceIntervalEvents events should only be used internally in IntervalCollections.
```

Reply:
or is this prohibited by api-extractor config? didn't look closely at usage (and IIRC it's somewhat config dependent what it'll let you do in this realm)

Reply:
It does cause an api-extractor warning.

Reply:
sounds good, as-is is fine then.